sample/Gemfile

```diff
@@ -6,3 +6,4 @@ ruby ">= 2.6.10"
 gem 'cocoapods', '1.16.2'
 gem 'cocoapods-check', '1.1.0'
 gem 'activesupport', '>= 6.1.7.5', '< 7.3.0'
+gem 'xcpretty'
```

Comment:
The `test_ios` script uses `xcpretty`. Adding it here so that it's installed when you run `yarn pod-install` from the root.